crd-convert/src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,11 @@
+name = WebhookConfig
```

Comment:
Do we still need this when we don't have the webhook?

Reply:
I guess not? Removing ...